internal/chain/ws_client.go

```diff
@@ -65,6 +65,17 @@ func (client *WsClient) GetTransactionByHash(hash string) (trx Transaction, err
 	return
 }
 
+func (client *WsClient) GetTransactionLogsByHash(hash string) (logs []EventLog, err error) {
```

Comment:
why to add method that won't be really used? 
You can use `GetTransactionByHash` in tests and implement in mock client